lib/livebook/teams.ex

```diff
@@ -37,6 +38,43 @@ defmodule Livebook.Teams do
     create_org_request(org, attrs, &Requests.join_org/1)
   end
 
+  @doc """
+  Subscribes to one or more subtopics in `"teams"`.
+
+  ## Messages
+
+  Topic `teams:deployment_groups`:
+
+    * `{:deployment_group_created, DeploymentGroup.t()}`
+    * `{:deployment_group_updated, DeploymentGroup.t()}`
+    * `{:deployment_group_deleted, DeploymentGroup.t()}`
+
+  """
+  @spec subscribe(atom() | list(atom())) :: :ok | {:error, term()}
+  def subscribe(topics) when is_list(topics) do
+    for topic <- topics, do: subscribe(topic)
+
+    :ok
+  end
+
+  def subscribe(topic) do
+    Phoenix.PubSub.subscribe(Livebook.PubSub, "teams:#{topic}")
+  end
+
+  @doc """
+  Unsubscribes from `subscribe/0`.
+  """
+  @spec unsubscribe(atom() | list(atom())) :: :ok
+  def unsubscribe(topics) when is_list(topics) do
+    for topic <- topics, do: unsubscribe(topic)
+
+    :ok
+  end
+
+  def unsubscribe(topic) do
+    Phoenix.PubSub.unsubscribe(Livebook.PubSub, "teams:#{topic}")
+  end
```

Comment:
Let's move subscribe and unsubscribe to the Broadcasts module. I have already done the same for Hubs.Broadcasts in main.